### PR TITLE
ROX-18698: Add more logs when sensor is disconnecting

### PR DESCRIPTION
**Pull request by fredrb**
## Description

This will help with future debugging when sensor frequently disconnects, we can more easily asses what triggered it. 

## Checklist
- [ ] Investigated and inspected CI test results
- [ ] ~~Unit test and regression tests added~~
- [ ] ~~Evaluated and added CHANGELOG entry if required~~
- [ ] ~~Determined and documented upgrade steps~~
- [ ] ~~Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))~~

If any of these don't apply, please comment below.

### Here I tell how I validated my change

There's no real validation until we get logs from users cluster. 

What can be validated here is: check the info messages that show up in sensor logs once prow finishes running E2E tests. 

### Reminder for reviewers

In addition to reviewing code here, reviewers **must** also review testing and request further testing in case the
performed one does not seem sufficient. As a reviewer, you must not approve the change until you understand the
performed testing and you are satisfied with it.

### CODE REVIEW

File: sensor/common/sensor/central_receiver_impl.go [hunk 1]
```diff
@@ -39,19 +39,23 @@ func (s *centralReceiverImpl) receive(stream central.SensorService_CommunicateCl
 	for {
 		select {
 		case <-s.stopper.Flow().StopRequested():
+			log.Infof("Central receiver stop flow requested")
```
Comment (vikin91) [inline sensor/common/sensor/central_receiver_impl.go:42]: I thought this is a typo and wanted to correct `receiver` -> `received`.
Maybe:

```suggestion
			log.Infof("Central receiver got stop flow requested")
```

**Review by vikin91 — COMMENTED** [review]
Great idea, but let's make the log messages another look to make sure that they read well

File: sensor/common/sensor/central_sender_impl.go [hunk 2]
```diff
@@ -81,12 +81,15 @@ func (s *centralSenderImpl) send(stream central.SensorService_CommunicateClient,
 		select {
 		case msg, ok = <-componentMsgsC:
 			if !ok {
+				log.Infof("Central sender stopping due to channel closing")
```
[the remark this answers is not in the document]
Comment (fredrb) [reply sensor/common/sensor/central_sender_impl.go:84]: I thought only the package name was included, but just opening any logs I see that filenames are also included. I'll update the messages to remove the prefix

**Comment by fredrb** [comment]
@vikin91 please check V2 :-)

**Review by vikin91 — APPROVED** [review]
Thanks for improving!

**Pull request merged**